piker/data/feed.py

```diff
@@ -335,6 +355,31 @@ async def attach_feed_bus(
             bus._subscribers[symbol].remove(sub)
 
 
+@asynccontextmanager
+async def open_sample_step_stream(
```

Comment:
Allows us to actor-cache the OHLC step event stream per delay (since you likely will want the same event for all local consumers.